tests/test_renault.py

```diff
@@ -10,7 +10,7 @@ def test_renault():
     assert vin.manufacturer == 'Renault'
     assert vin.vds == '4SRAP4'
     assert vin.vis == '51234567'
-    assert vin.years == [2005]
+    assert vin.years == [2035, 2005]
```

Comment:
I couldn't think of a legitimate reason to have years in future here.
Do you have any?

Reply:
I find it annoying that the result changes depending on the current year (unit tests can break just after New Year's Eve :)).

> On April 30, 2008, the US National Highway Traffic Safety Administration adopted a final rule amending 49 CFR Part 565, "so that the current 17 character vehicle identification number (VIN) system, which has been in place for almost 30 years, can continue in use for at least another 30 years"

https://en.wikipedia.org/wiki/Vehicle_identification_number#Model_year_encoding
https://www.law.cornell.edu/cfr/text/49/part-565

This means that it will be valid until at least 2038.

So I think `years` should be kept simple and turn over all possible years. Then we can think of a `most_likely_year` property that will pick the most likely year (depending on a strategy)?

Reply:
> (unit tests can break just after New Year's Eve :)).

Can you give an example?

> This means that it will be valid until at least 2038.

Techically, but for a person using this library checking some non synthetic Vin would seem like a journey to the future. It seems like a wierd.

Reply:
For example `assert vin.years == [1992]` is correct on 31 December 2021 but becomes incorrect on 1 January 2022 ([2022, 1992])

Reply:
Ah, that's what I thought. Those test have 14 years to function %)
That said those sould rather be fixed to something like 
```
assert min(vin.years) == 2005
```